idd/Energy+.idd.in

```diff
@@ -71830,8 +71831,7 @@ HeatPump:PlantLoop:EIR:Heating,
         \note sizing the heat pump as well as to allow checks for unexpected
         \note simultaneous operation of the two objects.
         \type object-list
-        \object-list WWHPCoolingNames
-        \reference PLHPCoolingNames
+        \object-list PLHPCoolingNames
```

Comment:
And Heating references Cooling

Reply:
So this was an object-list type but didn't actually have an object-list defined.  Nice fix.